pkg/controllers/user/pipeline/controller/project/project.go

```diff
@@ -0,0 +1,99 @@
+package project
```

Comment:
@gitlawr we should put a description to each and every controller declaring what a particular controller does. At the beginning of the file